src/Module/Api/Friendica/Photo/Delete.php

```diff
@@ -32,7 +32,7 @@
  */
 class Delete extends BaseApi
 {
-	protected function rawContent(array $request = [])
+	protected function post(array $request = [], array $post = [])
```

Comment:
Why are we not supporting DELETE anymore?

Reply:
I don't see a sense in supporting multiple methods. I also saw this when comparing our implementation with the Twitter API. Twitter only supports `POST` and `GET`. This is in contrast to the Mastodon API. But there the same endpoint reacts to different methods in a different way.

Reply:
One sense is that we used to support different methods, so it's a backward-breaking change. In the router config file the deletion endpoint can be assigned several methods without the module having to know or care about it, so I think we should take advantage of it.

Reply:
I do prefer a breaking change here in favour of cleaning up stuff. Eventually we should then add multiple methods (`GET` for fetching, `POST` for creating, `DELETE` for deleting, `PATCH` for updating) to endpoints like `friendica/photo`, `friendica/group` or `friendica/photoalbum` and deprecate the other endpoints.

Reply:
I'm not sure what this removal is cleaning, keeping DELETE support is just about adding a constant to the method array in the routes configuration file, the module code itself doesn't change. I like the idea of giving as much possibilities to API clients. Remember Postel's Law that you have cited before about Friendica? "Be conservative in what you send; be liberal in what you accept." This is a good example of this.

Reply:
I'm mostly convinced that the number of people using that part of the API is rather limited - and using it with `DELETE` instead of `POST` is surely even more smaller. And having a look at the documentation at `/help/API-Friendica` we see a wild mixture of different methods. I really would like to unify this. Of course we should then change the documentation as well, so that people know what method to use.